requirements/base.in

```diff
@@ -1,3 +1,5 @@
+code-annotations    # Parse feature toggle and setting annotations
+Django>=2.2,<2.3    # Version coherent with other projects
```

Comment:
I'm not clear why this needs to be added to base.in? If we need the constraint for some reason, should we add it to a new constraints.txt file?

Reply:
I guess that if we don't add the constraint, the most recent Django will be installed. I did not take the time to add a constraints file because there are already pinned requirements in this base.in file (see below).